app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -138,29 +133,6 @@ class HomeFragmentPresenter @Inject constructor(
     // If there's an error loading the data, assume the default.
     return Transformations.map(topicListSummaryResultLiveData) { it.getOrDefault(TopicList.getDefaultInstance()) }
   }
-
-  private fun subscribeToUserAppHistory() {
-    getUserAppHistory().observe(fragment, Observer<UserAppHistory> { result ->
-      userAppHistoryViewModel = UserAppHistoryViewModel()
```

Comment:
Also, rename `UserAppHistoryViewModel` to `WelcomeViewModel`

And correspondingly in `welcome.xml` file update that same. Also, in `welcome_text_view` the text value can directly be `Welcome to Oppia`

Reply:
Done :)